lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -202,16 +201,9 @@ def self.run_release_upload(params)
                 app_name: app_name
               )
 
-              destination_excludes_array = destination_excludes.split(',').map(&:strip).map(&:downcase)
               UI.abort_with_message!("Failed to list distribution groups for #{owner_name}/#{app_name}") unless distribution_groups
-              distribution_groups.each do |group|
-                destinations_array << group['name'] unless destination_excludes_array.include? group['name'].downcase 
-              end
-
-              distribution_group_names = distribution_groups.map {|h| h['name'].downcase }
-              destination_excludes_array.each do |group|
-                UI.important("Could not find the excluded distribution group: #{group}") unless distribution_group_names.include? group.downcase 
-              end
+              
+              destination_array = distribution_groups.map {|h| h['name'].downcase }
```

Comment:
I don't think we should downcase here (it apparently works but I wouldn't count on it)

Reply:
I agree, so I have removed it